firestore/integration_test_internal/src/android/cancellation_token_source.h

```diff
@@ -1,3 +1,5 @@
+// Copyright 2021 Google LLC
+
 #ifndef FIREBASE_FIRESTORE_CLIENT_CPP_SRC_TESTS_ANDROID_CANCELLATION_TOKEN_SOURCE_H_
 #define FIREBASE_FIRESTORE_CLIENT_CPP_SRC_TESTS_ANDROID_CANCELLATION_TOKEN_SOURCE_H_
```

Comment:
Lint: `Lines should be <= 80 characters long`<hidden value="cpplint-file-comment"></hidden><hidden value="firestore/integration_test_internal/src/android/cancellation_token_source.h:3:  Lines should be &lt;= 80 characters long  [whitespace/line_length] [2]"></hidden>